src/shared/inc/stresslog.h

```diff
@@ -543,25 +543,13 @@ struct StressMsg
     static const size_t formatOffsetLowBits = 26;
     static const size_t formatOffsetHighBits = 13;
 
-    union
-    {
-        struct
```

Comment:
Does removing this still allow you to support the older 7.0/6.0 runtimes?

Reply:
Yes, the old runtime support is through this function: https://github.com/dotnet/diagnostics/pull/3779/files#diff-692bcdb15f7c064bb2422b6497b1c72301235b3736df0cd42fb4213c2dab3cc3R335

Since the messages are the same size, I basically type-pun them and translate from old to new.

I do need to make one more change though, as I just realized that my implementation might break if you try to `!dumplog` twice, which isn't an absurd thing to do.

Reply:
Looks like the stresslog is read from the remote process into a new buffer every time `!dumplog` is run, so this wouldn't break running it twice.